tests/refs/transactions.c

```diff
@@ -108,3 +108,24 @@ void test_refs_transactions__unlocked_set(void)
 	cl_git_fail_with(GIT_ENOTFOUND, git_transaction_set_target(g_tx, "refs/heads/foo", &id, NULL, NULL));
 	cl_git_pass(git_transaction_commit(g_tx));
 }
+
+void test_refs_transactions__error_on_locking_locked_ref(void)
+{
+	git_oid id;
+	git_transaction *g_tx_with_lock;
+	git_repository *g_repo_with_locking_tx;
+	const char *g_repo_path = git_repository_path(g_repo);
+	
+	// prepare a separate transaction in another instance of testrepo and lock master
```

Comment:
We're trying to be as C90 compatible as possible, thus you should use `/* */` style comments